src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (let i = 0; i < rows.length; i++) {
+  const row = rows[i];
+
+  const clonedCell = row.cells[1].cloneNode(true);
```

Comment:
This line will throw an error if the row doesn't have at least two cells because `row.cells[1]` will be `undefined`. It's recommended to add a condition to check if the row has at least two cells before cloning a cell. For example, you can add `if (row.cells.length > 1)` before this line.